production/grafanacloud-install.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+# grafanacloud-install.sh installs the Grafana Cloud Agent on supported x86_64
+# Linux systems for Grafana Cloud users. Those who aren't users of Grafana Cloud 
+# or need to install the Agent on a different architecture or platform should 
+# try another installation method.
+#
+# grafanacloud-install.sh has a hard dependency on being run on a supported
+# Linux system. Currently only systems that can install deb or rpm packages 
+# are supported. The target system will try to be detected, but if it cannot, 
+# PACKAGE_SYSTEM can be passed as an environment variable with either rpm or 
+# deb.
+set -euo pipefail
+
+log() {
+  echo "$@" >&2 
+}
+
+fatal() {
+  log "$@"
+  exit 1
+}
+
+#
+# REQUIRED environment variables. 
+#
+GCLOUD_STACK_ID=${GCLOUD_STACK_ID:=} # Stack ID where integrations are installed
+GCLOUD_API_KEY=${GCLOUD_API_KEY:=}   # API key to communicate to the integrations API
+
+[ -z "$GCLOUD_STACK_ID" ] && fatal "Required environment variable $$GCLOUD_STACK_ID not set."
```

Comment:
```suggestion
[ -z "$GCLOUD_STACK_ID" ] && fatal "Required environment variable \$GCLOUD_STACK_ID not set."
```
Based on the assumption that you want the verbatim output `Required environment variable $GCLOUD_STACK_ID not set.`. `$$` returns the PID.

Reply:
Ha, thanks :)